lighthouse-extension/app/src/popup.js

```diff
@@ -134,10 +134,15 @@ document.addEventListener('DOMContentLoaded', _ => {
     })
     .catch(err => {
       let {message} = err;
```

Comment:
I believe this is correct.
`let {message} = err;`is equivalent to `let message = err.message;`